### ISSUE TITLE
Add anonymous to classmates mode

### ISSUE THREAD

**Issue by mterwill** [mterwill/classroom-help-queue#55]
It's been brought to my attention that student names/classes they are taken is private information. Students should be able to hide their names from other students.

**Comment by mterwill** [comment]
FERPA gives the following guidelines, which I believe are applicable here ([source](https://www2.ed.gov/policy/gen/guid/fpco/ferpa/index.html)):

> Schools may disclose, without consent, "directory" information such as a student's name, address, telephone number, date and place of birth, honors and awards, and dates of attendance. However, schools must tell parents and eligible students about directory information and allow parents and eligible students a reasonable amount of time to request that the school not disclose directory information about them.

I think we can implement this as an extension of the per-queue private mode added in https://github.com/mterwill/office-hours-help-queue/pull/161. Students can toggle a setting that will hide their information from other students on the queue.

**Comment by developStorm** [comment]
@mterwill You're right about name being part of directory information. However, enrollment information such as which class a student is in is not considered directory information and should not be disclosed without consent (OH queue effectively leaks this). Additionally, students have the right under FERPA to restrict the release of their directory information, which would include their name.

It's crucial to ensure that our implementation fully complies with FERPA regulations and protects student privacy. I don't think it makes sense to post students' names publicly in an oh queue context, it may also make students feel too embarrassed to come up and ask questions etc. This information should simply stop being made public IMHO.

Also, privacy@umich.edu might be a good resource to consult for this issue.
